spec/schemas/_common.yaml

```diff
@@ -2152,31 +2152,31 @@ components:
           type: integer
           format: int32
         total:
-          description: The number of documents that were successfully processed.
+          description: The number of documents that were successfully processed. 
           type: integer
           format: int64
         updated:
-          description: The number of documents that were successfully updated, for example, a document with same ID already existed prior to reindex updating it.
+          description: The number of documents that were successfully updated, for example, a document with same ID already existed prior to the reindex updating it. 
```

Comment:
This sentence needs some revision. The first part of the sentence doesn't connect syntactically to the example.